### PR TITLE
Global View: Fix Themes and Plugins/manage navigation

### PR DESCRIPTION
**Pull request by BogdanUngureanu**
For Single site users the Theme Showcase and Plugins > Manage used in the global view were pointing to the in-site context UI.

This PR makes it consistent with the other UIs (Domains, Plugins) and loads the Global View version.

Related to https://github.com/Automattic/wp-calypso/issues/96619

## Proposed Changes

* Point `/themes` to the Global View version for single site users
* Point `/plugins/manage` to the Global View version for single-site users

## Why are these changes being made?

* Consistency: Sites, Domains and Plugins are all pointing to the global view.

## Testing Instructions

* Use the live link and open an incognito window
* Login with a test account that has only one site
* Go to /sites and click on "Themes" and "Plugins > Manage Plugins"
* Make sure the Global View is now loaded instead of the in-site one

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
    - [ ] For UI changes, have we tested the change in various languages (for example, ES, PT, FR, or DE)? The length of text and words vary significantly between languages. 
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/my-sites/themes/atomic-transfer-dialog.tsx [hunk 1]
```diff
@@ -107,11 +107,21 @@ class AtomicTransferDialog extends Component< AtomicTransferDialogProps > {
...
 			this.continueToActivate();
 		}
 
-		if ( siteId && uploadError && prevProps.uploadError !== uploadError ) {
+		if (
+			siteId &&
+			uploadError &&
+			prevProps.uploadError !== undefined &&
+			prevProps.uploadError !== uploadError
+		) {
```
Comment (mmtr) [inline client/my-sites/themes/atomic-transfer-dialog.tsx:124]: Are these changes related to this PR?

**Review by mmtr — APPROVED** [review]

File: client/my-sites/themes/atomic-transfer-dialog.tsx (hunk 1 above)
Comment (BogdanUngureanu) [reply client/my-sites/themes/atomic-transfer-dialog.tsx:124]: Yes. Calypso crashed because of a JS error when a user with a single site was clicking on any theme from the theme showcase (from the Global View).
